packages/flutter_tools/lib/src/build_system/targets/web.dart

```diff
@@ -604,7 +604,7 @@ class WebServiceWorker extends Target {
       <String>[
         'main.dart.js',
         'index.html',
-        if (urlToHash.containsKey('assets/AssetManifest.json'))
+        if (urlToHash.containsKey('assets/AssetManifest.bin.json'))
           'assets/AssetManifest.json',
```

Comment:
Just to double check, the key you are checking for isn't the same string you are adding to the list?

Reply:
Good grief, that's embarrassing. Fixed.

Reply:
All good, everything else LGTM! 👍